app/lib/common/widgets/typing_indicator.dart

```diff
@@ -45,11 +56,15 @@ class TypingWidget extends StatelessWidget {
   }
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     final l10n = L10n.of(context);
+    final typingUsers = _getTypingUsers(ref);
     final text = _buildTypingText(typingUsers, l10n);
 
+    if (typingUsers.isEmpty) return const SizedBox.shrink();
```

Comment:
I think this can be simplified, no?

```suggestion
    if (ref.watch(chatTypingEventProvider(roomId)).valueOrNull?.isNotEmpty != true) return const SizedBox.shrink();
    
    
    final text = _buildTypingText(typingUsers, l10n);
```